Formula/uwsgi.rb

```diff
@@ -31,6 +31,13 @@ class Uwsgi < Formula
     sha256 "d71c879774b32424b5a9051ff47d3ae6e005412e9214675d806857ec906f9336"
   end
 
+  # Fix "library not found for -lgcc_s.10.5" with 10.14 SDK
+  # Remove in next release
+  patch do
```

Comment:
Should this be in a `stable do` block?

Reply:
That's why I hate `head`… keep moving things around for no good reason :(